forms/TreeDropdownField.php

```diff
@@ -337,9 +337,8 @@ public function tree(SS_HTTPRequest $request) {
 		}
 
 		// main node leaf limit configuration
-		$nodeCountThreshold = 30;
```

Comment:
You may want a different threshold for treedropdown field than you might use for, say, the sitetree hierarchy view.